test/ui/ui-main.poolsLiquidity.addRemove.test.ts

```diff
@@ -0,0 +1,226 @@
+/*
+ *
+ * @group ui
+ */
+import { Mangata } from "../../utils/frontend/pages/Mangata";
+import { Keyring } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+import { WebDriver } from "selenium-webdriver";
+import { getApi, initApi } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { Sidebar } from "../../utils/frontend/pages/Sidebar";
+import { DriverBuilder } from "../../utils/frontend/utils/Driver";
+import {
+  addExtraLogs,
+  setupPolkadotExtension,
+} from "../../utils/frontend/utils/Helper";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { AssetWallet, User } from "../../utils/User";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import { MGA_ASSET_ID, MGR_ASSET_NAME } from "../../utils/Constants";
+import { ExtrinsicResult, waitNewBlock } from "../../utils/eventListeners";
+import {
+  createAssetIfMissing,
+  createPoolIfMissing,
+  mintLiquidity,
+} from "../../utils/tx";
+import { SudoUser } from "../../utils/Framework/User/SudoUser";
+import { Node } from "../../utils/Framework/Node/Node";
+import { connectPolkadotWallet } from "../../utils/frontend/utils/Handlers";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+import { Pool } from "../../utils/frontend/pages/Pool";
+import { BurnLiquidityModal } from "../../utils/frontend/pages/BurnLiquidityModal";
+import {
+  ModalType,
+  NotificationModal,
+} from "../../utils/frontend/pages/NotificationModal";
+import { Polkadot } from "../../utils/frontend/pages/Polkadot";
+
+require("dotenv").config();
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+
+jest.setTimeout(1500000);
+let driver: WebDriver;
+let sudo: SudoUser;
+let testUser1: User;
+const testAssetName = "TST4";
+let testAssetId: BN;
+
+describe("UI tests - adding, removing liquidity", () => {
+  beforeAll(async () => {
+    try {
+      getApi();
+    } catch (e) {
+      await initApi();
+    }
+
+    const keyring = new Keyring({ type: "sr25519" });
+    const node = new Node(getEnvironmentRequiredVars().chainUri);
+    await node.connect();
+    sudo = new SudoUser(keyring, node);
+    keyring.addPair(sudo.keyRingPair);
+    await setupApi();
+    await setupUsers();
+    driver = await DriverBuilder.getInstance();
+    const { mnemonic } = await setupPolkadotExtension(driver);
+    testUser1 = new User(keyring);
+    testUser1.addFromMnemonic(keyring, mnemonic);
+    testAssetId = await createAssetIfMissing(sudo, testAssetName);
+    await createPoolIfMissing(
+      sudo,
+      "1800000000000000000000000000000000",
+      MGA_ASSET_ID,
+      testAssetId
+    );
+
+    await Sudo.batchAsSudoFinalized(
+      Assets.mintToken(new BN(7), testUser1), // transferAll test
```

Comment:
7 ? why this one?

Reply:
ha :D not needed, just an oversight, fixing